app/views/layouts/base.html.erb

```diff
@@ -44,7 +44,7 @@
       document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
     </script>
 
-    <%= render CookieBannersComponent.new(current_namespace: try(:current_namespace), request_path: request.path) %>
+    <%= render CookieBannersComponent.new(current_namespace: try(:current_namespace), request_path: request.path, current_cookies: cookies['consented-to-apply-cookies']) %>
```

Comment:
I think this layout is used for Manage and Apply, so it would be good to cover both bases. Maybe we could pass in the whole cookies hash and then inside the component have a list like

```ruby
RELEVANT_COOKIES = %w[consented-to-apply-cookies consented-to-manage-cookies].freeze
```

and then check if `@cookies.keys` includes those? This would also make the interface asking for `current_cookies` better because the caller doesn't need to know which cookies are relevant, it can literally pass _the current cookies_ (if that makes sense).

Reply:
Thanks makes sense! `.keys` wasn't available to the object so I converted to a hash - a bit messy. It has `key?` but would only work if checking one string object